schema/schema.cc

```diff
@@ -1382,6 +1382,17 @@ schema_ptr schema_builder::build(schema::raw_schema& new_raw) {
         }
     }
 
+    if (static_props.is_group0_table) {
+        new_raw._extensions[tombstone_gc_extension::NAME] = ::make_shared<tombstone_gc_extension>(
+            tombstone_gc_options({{"mode", "repair"},
+                                  // We don't need the propagation delay for the group0 tables:
```

Comment:
#include "my standard rant about this.h"

Please address this.

Reply:
What is there to address? Can you please reply to this https://github.com/scylladb/scylladb/pull/20394#discussion_r1744964340  and do not simple abandon a conversation in the middle?

Reply:
Changed to handle the group0 tables based on the schema property (removed switching to the "repair" mode)